cmd/root_test.go

```diff
@@ -69,11 +84,7 @@ func newGlobalTestState(t *testing.T) *globalTestState {
 
 	outMutex := &sync.Mutex{}
 	defaultFlags := getDefaultFlags(".config")
-
-	// Set an empty REST API address by default so that `k6 run` dosen't try to
-	// bind to it, which will result in parallel integration tests trying to use
-	// the same port and a warning message in every one.
-	defaultFlags.address = ""
+	defaultFlags.address = fmt.Sprintf("localhost:%d", getFreePort(t))
```

Comment:
same nitpick here, could be worth (but not necessary) using `net.JoinHostPort` here for the sake of compatibility.

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/a6823b208ee3353e499354e2fc5f2fe6dcb61dc8..b37b0dd69a8f5e11806e202e001f2d490452e5a9), discussion in https://github.com/grafana/k6/pull/2789#discussion_r1037105698